### DISCUSSION TITLE
announcing

### DISCUSSION THREAD

**Discussion by kambiumdt** [music-assistant/support#1163] [Q&A] [answered]
In "old" version that have been work perfect.

In new version  gona be the same? :)

**Comment by OzGav** [comment, marked as answer]
If you mean text to speech (TTS) messages then yes but it will be a couple of weeks until the HA integration (the connection between MA and HA) is done

**Reply by kambiumdt** [reply]
great!
so thanks for your hard work, and i`m still waiting for "the never endind store with HA :)"

**Reply by kambiumdt** [reply]
i see, that its working now :)

Its possible to disable that "dzingle" before TTS?

**Reply by OzGav** [reply]
What playback device are you using?

**Reply by kambiumdt** [reply]
google mini, and lenovoclock

**Reply by OzGav** [reply]
For the Google mini I think that tone is a Google thing. Have a look in the Home app and see if you can disable it there. Not sure about the clock might be the same?

**Reply by OzGav** [reply]
I MA1 there was a bell then the tone which is why I think the tone is a Google thing…?

**Reply by kambiumdt** [reply]
i have disable the chime in google app.

Its only played when use "mass_player"

**Reply by marcelveldt** [reply]
Its on the roadmap for V2 to make that configurable and skip the announcement bell
